packages/zero/src/sqlite.ts

```diff
@@ -0,0 +1 @@
+export * from '../../replicache/src/sqlite.ts';
```

Comment:
Does this need a new package.json export to expose?

Reply:
Yes it does. Good catch